jigasi/rootfs/etc/cont-init.d/10-config

```diff
@@ -10,3 +10,27 @@ fi
 
 mkdir -pm777 /tmp/transcripts
 chown jigasi:jitsi /tmp/transcripts
+
+# Create Google Cloud Credentials
+if [[ $ENABLE_TRANSCRIPTIONS -eq 1 || $ENABLE_TRANSCRIPTIONS == "true" ]] && [[ ! -f /config/key.json ]]; then
+    jq -n \
+        --arg GC_PROJECT_ID "$GC_PROJECT_ID" \
```

Comment:
souldn't we check for the existence of all these env vars before building the json file and error if any is missing?

Reply:
I've added a check for them now.